recipes/ldas-tools-cmake/bld.bat

```diff
@@ -0,0 +1,26 @@
+setlocal EnableDelayedExpansion
```

Comment:
Why this is necessary?

Reply:
perhaps its not? I have removed that line, so lets see what happens...